### PR TITLE
feat: add failed screenshot diffs status

### PR DESCRIPTION
**Pull request by jsfez**
Capture d’écran 2022-09-08 à 10 35 57

### CODE REVIEW

File: apps/graphql/src/definitions/ScreenshotDiff.js [hunk 1]
```diff
@@ -53,10 +54,14 @@ export const resolvers = {
         expiresIn: 7200,
       });
     },
-    status(screenshotDiff) {
+    async status(screenshotDiff) {
       switch (screenshotDiff.score) {
-        case null:
-          return "added";
+        case null: {
+          const { name } = await screenshotDiff.$relatedQuery(
```
[the remark this answers is not in the document]
Comment (gregberge) [reply apps/graphql/src/definitions/ScreenshotDiff.js:60]: Yeah, you have to use a `data-loader` for that.

**Review by gregberge — CHANGES_REQUESTED** [review]

File: apps/app/src/pages/Build/ScreenshotDiffStatusIcons.js [hunk 2]
```diff
@@ -3,12 +3,15 @@ import {
...
   FileRemovedIcon,
+  FileIcon,
 } from "@primer/octicons-react";
 
 export function ScreenshotDiffStatusIcon(type) {
   switch (type) {
     case "added":
       return FileAddedIcon;
+    case "failed":
+      return FileIcon;
```
Comment (gregberge) [inline apps/app/src/pages/Build/ScreenshotDiffStatusIcons.js:14]: Weird.

File: apps/app/src/pages/Build/index.js [hunk 3]
```diff
@@ -363,16 +368,25 @@ const BuildContent = ({ ownerLogin, repositoryName, buildNumber }) => {
...
           ) : null}
 
+          {diffGroups.failed.length > 0 && (
+            <>
+              <SecondaryTitle mt={6} color={getStatusPrimaryColor("danger")}>
+                Failed Screenshots
+              </SecondaryTitle>
+              <ScreenshotCards screenshotDiffs={diffGroups.failed} />
+            </>
+          )}
```
Comment (gregberge) [inline apps/app/src/pages/Build/index.js:378]: Time to create a component to factorize?

**Review by gregberge — CHANGES_REQUESTED** [review]

File: apps/app/src/pages/Build/ScreenshotDiffStatusIcons.js (hunk 2 above)
Comment (jsfez) [reply apps/app/src/pages/Build/ScreenshotDiffStatusIcons.js:14]: I didn't find a more suitable "file" icon in Octicons lib. 
In an other hand we can give up "file icons" and use icons like "x-circle-fill"

https://primer.style/octicons/x-circle-fill-24

**Review by gregberge — APPROVED** [review]

**Pull request merged**